xs/src/libslic3r/MultiPoint.hpp

```diff
@@ -31,6 +31,25 @@ class MultiPoint
 
     int find_point(const Point &point) const;
     bool has_boundary_point(const Point &point) const;
+	/// return the index of the closest point in this polygon in relation with "point"
+	/// \param point the point to compare with. 
+	/// \return the index of the closest point in the points vector. 
+    size_t closest_point_index(const Point &point) const {
+        size_t idx = -1;
```

Comment:
-1 into unsigned

Reply:
How, yes. I copy this and then decided to replace int by size_t.
Well, i didn't see the problem. we can consider size_t max value as "incorrect".
I'll add some comment on the dioxygen.

Reply:
https://en.cppreference.com/w/cpp/header/limits